processing-service-container/src/test/java/com/github/cafdataprocessing/processing/service/tests/GlobalConfigIT.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright 2015-2017 EntIT Software LLC, a Micro Focus company.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.github.cafdataprocessing.processing.service.tests;
+
+import com.github.cafdataprocessing.processing.service.client.ApiClient;
+import com.github.cafdataprocessing.processing.service.client.ApiException;
+import com.github.cafdataprocessing.processing.service.client.api.GlobalConfigApi;
+import com.github.cafdataprocessing.processing.service.client.model.GlobalConfig;
+import com.github.cafdataprocessing.processing.service.client.model.GlobalConfigsEntry;
+import com.github.cafdataprocessing.processing.service.tests.utils.ApiClientProvider;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import static org.junit.Assert.*;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+public class GlobalConfigIT
+{
```

Comment:
Add a test to check that update works e.g. create a config, retrieve it to check its values then call the set method for same key with a new default value and description and then retrieve it to verify it has been updated.